src/services/ProcessorServiceProjectMember.js

```diff
@@ -27,8 +27,9 @@ function createIdSchema () {
 function updateSchema () {
   return createIdSchema().keys({
     isPrimary: Joi.boolean(),
-    role: Joi.any().valid(PROJECT_MEMBER_ROLE.CUSTOMER, PROJECT_MEMBER_ROLE.MANAGER,
-      PROJECT_MEMBER_ROLE.ACCOUNT_MANAGER, PROJECT_MEMBER_ROLE.COPILOT, PROJECT_MEMBER_ROLE.OBSERVER).required()
+    // unlike in Project Service endpoints, in this processor we should let create members with any roles
+    // because Project Service may create members with any roles implicitly, when accepting invitations
+    role: Joi.string().valid(_.values(PROJECT_MEMBER_ROLE)).required()
```

Comment:
@maxceem can it create any security loop hole? I mean if some one is able to raise the bus event with data which is not allowed by the project service, then if we don't have checks here, it might get through. However, on second thoughts, I think sending events to bus is not possible without authentication so we should be safe. What do you think?

Reply:
In my opinion, this place doesn't bring an additional security loop. If someone happens to be able to send arbitrary messages to the Bus Api or Kafka, he could do much bigger harm than adding a member: removing or updating projects or any other arbitrary data which we store in ES. Also, he can still update or remove members.

We may still think if there is any chance someone could gain access to the Bus Api or Kafka because if so, it gives almost full control over the data we have inside ES.

Reply:
Agreed.